cloud/kyup/kyup.py

```diff
@@ -0,0 +1,338 @@
+#!/usr/bin/python -t
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: kyup
+short_description: create, terminate, start or stop a container in kyup.com, return container_id.
+description:
+    - Creates or terminates Kyup container. When created optionally waits for it to be 'running'. This module has a dependency on python-json and python-requests.
+version_added: "2.2"
+author: Marian Marinov <mm@kyup.com>
+options:
+  api_key:
+    description:
+      - Kyup.com API key. The key can also be supplied by setting the Environment variable KYUP_API_KEY.
+    required: true
+  enc_key:
+    description:
+      - Kyup.com encryption key. The key can also be supplied by setting the Environment variable KYUP_ENC_KEY.
+    required: false
+  action:
+    description:
+      - Action you want to perform on the container.
+    required: true
+  name:
+    description:
+      - Name of the container that you will create/operate on
+    required: true
+  ssh_keys:
+    description:
+      - Comma separated list of ssh key names, that have access to this container.
+    required: false
+  password:
+    description:
+      - Password for the root user of the container. Required when I(action) = 'create'.
+    required: false
+  image:
+    description:
+      - Template image to use when creating the container. Required when I(action) = 'create'.
+    required: false
+  dc_id:
```

Comment:
I found somewhere in the documentation that aliases should be removed from DOCUMENTATION and that is why I removed them